srcpkgs/mspds/template

```diff
@@ -0,0 +1,26 @@
+# Template file for 'mspds'
+pkgname=mspds
+version=3.15.1.1
+revision=1
+archs="x86_64 i686"
+makedepends="boost-devel hidapi-devel"
+depends="hidapi"
+short_desc="Texas Instrumets MSP Debug Stack, provider of libmsp430.so"
+maintainer="Benjamin Calderon <benj.calderon@gmail.com>"
+license="BSD-3-Clause"
+homepage="https://www.ti.com/tool/MSPDS"
+distfiles="https://dr-download.ti.com/software-development/driver-or-library/MD-4vnqcP1Wk4/3.15.1.1/MSPDebugStack_OS_Package_${version//./_}.zip"
+checksum=e3a59a98c43de7a92e5814d8c3304026165e6d2551e60acaca1f08c6b1a4bac8
+
+pre_build() {
+	ln -s ${XBPS_CROSS_BASE}/usr/include/hidapi/hidapi.h ThirdParty/include/
+}
+
+do_build() {
+	make ${makejobs}
+}
+
+do_install() {
+	vcopy ${wrksrc}/libmsp430.so /usr/lib
```

Comment:
You don't need to specify `${wrksrc}` here, you're already in that directory in this stage.